Buzz_gazebo/src/ROSBuzz.cpp

```diff
@@ -0,0 +1,17 @@
+
+#include <ros/ros.h>
+#include "BuzzBridge.hpp" 
+
+int main(int argc, char** argv)
+{
+  //  Initialize rosbuzz node
+  ros::init(argc, argv, "rosbuzz");
+  static ros::NodeHandle nodeHandle("/rosbuzz_node");
+  
+  // Initialize and start node execution
+  BuzzBridge bridge(&nodeHandle);
```

Comment:
See my comment on the getParam method in BuzzBridge.cpp line 21. 
The "/rossbuz_node" could be removed. This argument has the effect to add an additional string in all topics and services published by this node (/rosbuzz_node/topic_name). The other effect is that, if the declared name of the node in the launchfile is different from "/rosbuzz_node", the getParam() won't have the right path to the param and will failed. This is a problem if we want to declare a RosBuzz node for each robot. Overall, I think we should only use the non-overloaded constructor.

If we want to add a namespace before the topics it could be added in the launchfile instead.